tox.ini

```diff
@@ -6,6 +6,7 @@ skipsdist = true
 deps=
 	pytest
 	pytest-docker-db
+        pytest-freezegun
```

Comment:
looks like different indentation compared to the other?

Reply:
I missed that the other ones were tabbed, it still looks odd, I will fix this. should it be tabs?

Reply:
I think spaces are nicer